index.html

```diff
@@ -1,12 +1,15 @@
-
+<!DOCTYPE html>
 <!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+<!--[if gt IE 8]><!-->
```

Comment:
There are loads of checks for retro-compatibility but we are using CSS features that are not supported by these browsers. We could do a clean up of these tags.

Reply:
Sure we will clean this up :) 👍